src/auth/reset_password/reset_password.modal.js

```diff
@@ -10,7 +10,7 @@
         return {
             open: function () {
                 modal = ModalWindow(
-                    $compile('<motech-resetpassword />')($rootScope),
+                    $compile('<reset-password />')($rootScope),
                     $compile('<span>{{ "security.enterEmailQuestions" | translate}}</span>')($rootScope),
```

Comment:
Let's change the modal title from "Cant sign in? Forgot your password?" to "Reset Instructions" as it was previously made in Motech. It doesn't look good to repeat the same words at the beginning of the modal content and in the modal title.